resources/views/account/partials/queries/orders.graphql

```diff
@@ -14,7 +14,10 @@
             value
           }
         }
+        items {
+          product_name
```

Comment:
```suggestion
          product_name
          product_sku
          product_sale_price {
            value
          }
          quantity_ordered
```

To keep it inline with: https://github.com/rapidez/account/blob/master/resources/views/account/partials/queries/order.graphql

If this is going to change in the future maybe we should extract it to a partial so we don't have any duplication